reference/consent.v1.yaml

```diff
@@ -39,88 +55,207 @@ tags:
+            x - <to be defined>
+
+            99- Other, see statusMessage for details
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/consentrequest'
-      summary: ''
-  /consentrequest/{consentRequestId}:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  statusMessage:
+                    type: string
+                  consent:
+                    $ref: '#/components/schemas/Consent'
+        '':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Consent'
+      summary: Request Consent update
+    parameters: []
+  '/consent/{referenceId}':
     get:
       description: Resource to check status of requested consent
       tags:
         - consentrequest
       operationId: requestConsentStatus
-      parameters:
-        - name: consentRequestId
-          in: path
-          description: id of the ConsentRequest posted
-          schema:
-            type: string
-            format: uuid
-          required: true
       responses:
         '200':
-          description: OK, returned consentrequest can be checked for status 
+          description: 'OK, returned consentrequest can be checked for status'
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/consentrequest'
-        '404':
-          description: NOT FOUND, consentrequest unavailable
-      summary: ''
-  /consent:
+                $ref: '#/components/schemas/Consent'
+      summary: Get
+    parameters:
+      - schema:
+          type: string
+        name: referenceId
+        in: path
+        required: true
+        description: get consent information based on own referenceId
+  '/consents/{schoolid}':
     get:
-      description: Resource to check if a school (based on digideliveryid) has given consent to access data. This resource can be used to check if a school has given consent without a prior consentrequest from the application itself.
+      description: Resource to check if a school (based on schoolid) has given consent to access data. This resource can be used to check if a school has given consent without a prior consentrequest from the application itself.
       tags:
         - consent
-      operationId: getConsent
+      operationId: getConsents
       responses:
         '200':
-          description: OK, returned consentrequest can be checked for status 
+          description: 'OK, returned consentrequest can be checked for status'
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/consent'
-        '404':
-          description: NOT FOUND, consent unavailable
-      summary: ''
+                $ref: '#/components/schemas/Consent'
+      summary: Get Consents by schoolid
```

Comment:
Misschien moeten we nog iets opnemen dat het de verantwoordelijkheid is van de opvragende partij om alleen de consent-info te tonen aan degene van de school? Als het SIS op deze manier van elke school (zonder schoolid in het token toch?) de consent-info kan ophalen.